Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
 gem 'rack'
 gem 'sinatra'
-gem 'grape'
```

Comment:
Yeah I didn't think Grape was required of Sinatra or Rack either but I did run into an installation issue on another machine? (possibly the Ruby environment itself?). Happy to remove unless the issue comes up again